### PR TITLE
make indexOf function

### PR DESCRIPTION
**Pull request by mazurenko913**

### CODE REVIEW

File: src/array/indexOf/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function indexOf (array, value) {
```
Comment (vitaliymaz) [inline src/array/indexOf/index.js:1]: not ready

Comment (vitaliymaz) [reply src/array/indexOf/index.js:1]: add test when fromIndex is negative or bigger than array

File: src/array/indexOf/index.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+it('should return -1 if fromIndex is not a number', () => {
+  expect(indexOf([1, 2, 3], 'a')).toEqual(-1)
+})
+
+it('should return -1 when fromIndex bigger than array', () => {
+  expect(indexOf([1, 2, 3], 1, 5)).toEqual(-1)
+})
+
+it('should return -1 when fromIndex the element is not in the array', () => {
```
Comment (vitaliymaz) [inline src/array/indexOf/index.test.js:25]: 1. it() text has orthographic mistakes. Please use a spell check. The best option is Grammarly
2. I don't understand by the text what case should be tested here
3. 3-rd expect test something different than 1-st and 2-nd.
